src/custom/components/WalletModal/Option.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
```

Comment:
If no longer a mod, do we need this file?

Reply:
Its still a mod that has some small css changes.